src/dmd/expression.d

```diff
@@ -2018,7 +2018,7 @@ extern (C++) abstract class Expression : RootObject
             FuncDeclaration ff = outerfunc;
             if (sc.flags & SCOPE.compile ? ff.isPureBypassingInference() >= PURE.weak : ff.setImpure())
             {
-                error("`pure` %s `%s` cannot call impure %s `%s`",
+                error("`pure` %s `%s` cannot call im`pure` %s `%s`",
```

Comment:
I don't think highlighting parts of words improves readability. It just looks wrong.